conda/base/context.py

```diff
@@ -420,7 +415,10 @@ class Context(Configuration):
     dry_run = ParameterLoader(PrimitiveParameter(False))
     error_upload_url = ParameterLoader(PrimitiveParameter(ERROR_UPLOAD_URL))
     force = ParameterLoader(PrimitiveParameter(False))
-    json = ParameterLoader(PrimitiveParameter(False))
+    json = ParameterLoader(PrimitiveParameter(False, element_type=(bool, str)))
```

Comment:
I don't think this is necessary and probably overcomplicates json output. The user should instead just say `--console=alt-json`.

Reply:
I like this. Turning the the CLI option of `--json` to either be a boolean on/off or a option that accepts strings is going to be a pain! We would have to implement something custom ourselves to handle it.